docs/en/platform-engineers/definition-and-templates.md

```diff
@@ -234,3 +234,41 @@ spec:
 The specification of `schematic` is explained in following CUE and Helm specific documentations.
 
 Also, the `schematic` filed enables you to render UI forms directly based on them, please check the [Generate Forms from Definitions](openapi-v3-json-schema) section about how to.
+
+## Definition Revisions
+
+In KubeVela, definition entities are mutable. Each time a `ComponentDefinition` or `TraitDefinition` is updated, a corresponding `DefinitionRevision` will be generated to snapshot this change. Hence, KubeVela allows user to reference a specific revision of definition to declare an application.
+
+### Specify Definition Revision in Application
+
+For example, we can update the worker component definition by applying a new definition with same name as below.
+
+```shell
+kubectl apply -f https://raw.githubusercontent.com/oam-dev/kubevela/master/docs/examples/definition-revision/worker-v2.yaml
```

Comment:
please use a webservice as example because the above is webservice

Reply:
got it